### ISSUE TITLE
Addition of FAQ section on the official website of hvPlot website

### ISSUE THREAD

**Issue by DevanshRaizada** [holoviz/hvplot#1060]
The FAQ section is missing on the [hvPlot](https://hvplot.holoviz.org/index.html) library website and the addition of the section would be of great help to all the users

#### Is your feature request related to a problem? Please describe.
Basic questions like ''What objects can I use hvPlot with?'' can be answered very easily 

#### Describe the solution you'd like
A FAQ section covering all the fundamental question that comes to one's mind in the first place.

#### Additional context
An FAQ section is a vital section of any product website and is been missing on the hvPlot website.
If given a chance I would love to contribute on this issue.

**Comment by maximlt** [comment]
Hi @DevanshRaizada,

I'm not sure the documentation should have a FAQ. Some of the HoloViz websites have a FAQ and I'm not convinced about their usefulness, as they end up being just a grab bag of random stuff, that possibly wasn't well documented elsewhere, either because of the documentation structure or content.

Panel's documentation is being reworked and structured following the Diataxis framework. hvPlot documentation will have a similar fate, we don't have yet a timeline for that. A FAQ section is not part of the framework.

> An FAQ section is a vital section of any product website

I checked a few documentation sites, like from pandas, numpy, matplotlib, and saw no FAQ there. I guess there's a difference between more traditional product websites, and software documentation.

What would you like to find in an hypothetical hvPlot's FAQ?

**Comment by droumis** [comment]
I agree that a standalone FAQ page will probably not be included in the changes coming to the hvPlot website, but the information that would go in there should be readily accessible in the docs. Being tuned in to various forums, devs should have a sense of what the frequently asked questions are.

If there is some information that should be in the docs (or more clear/accessible), feel free to open an issue about it, even if you don't know where it should go. If you have a usage question that needs answering, the best place to ask is probably our Discord, where things get effectively upvoted as people find them more useful.
